### PR TITLE
Tjw secdir round2

### PR DESCRIPTION
**Pull request by moonshiner**
I added Term "DNS Administrator" 

I swapped out "foo" with "service" but I also did the `Service` thing in spots

but maybe "feature" is better than "service"

I suck at naming things

### CODE REVIEW

File: draft-ietf-dnsop-domain-verification-techniques.md [hunk 1]
```diff
@@ -198,6 +198,8 @@ Other techniques such as email or HTTP(S) based validation are out-of-scope.
 
 * `User`: the owner or operator of a domain in the DNS who needs to prove ownership of that domain to an Application Service Provider.
 
+* `DNS Adminstrator`: the owner or responsible party for the contents of a domain in the DNS.
```
Comment (enygren) [inline draft-ietf-dnsop-domain-verification-techniques.md:201]: Should we express the relationship to the User?  Perhaps put this first and extend User to append ", working in coordination with their DNS Administrator"

```suggestion
* `DNS Adminstrator`: the owner or responsible party for the contents of a domain in the DNS.

* `User`: the owner or operator of a domain in the DNS who needs to prove ownership of that domain to an Application Service Provider, working in coordination with their DNS Administrator.

```

**Review by enygren — COMMENTED** [review]
Looks good, but added one suggestion.  
Thanks also for putting the secdir comments into a file so we can update as we respond to them.

File: draft-ietf-dnsop-domain-verification-techniques.md (hunk 1 above)
Comment (moonshiner) [reply draft-ietf-dnsop-domain-verification-techniques.md:201]: Sure, I can see where that makes sense.  Let's add that

File: draft-ietf-dnsop-domain-verification-techniques.md [hunk 2]
```diff
@@ -196,7 +196,10 @@ Other techniques such as email or HTTP(S) based validation are out-of-scope.
 
 * `Validation Record`: the DNS record that is used to prove ownership of a domain name ({{RFC9499}}). It typically contains an unguessable value generated by the Application Service Provider which serves as a challenge. The Application Service Provider looks for the Validation Record in the zone of the domain name being verified and checks if it contains the unguessable value.
 
-* `User`: the owner or operator of a domain in the DNS who needs to prove ownership of that domain to an Application Service Provider.
+* `DNS Adminstrator`: the owner or responsible party for the contents of a domain in the DNS.
+
+* `User`: the owner or operator of a domain in the DNS who needs to prove ownership of that domain to an Application Service Provider, working in coordination with their DNS Administrator.
+
```
Comment (enygren) [inline draft-ietf-dnsop-domain-verification-techniques.md:202]:
```suggestion
```

**Review by enygren — APPROVED** [review]

**Pull request merged**